src/config/dynamic-theme-fixes.config

```diff
@@ -2898,32 +2898,83 @@ span[class^="burger"]
 codeforces.com
 
 INVERT
-#header > div:first-child img
-.action-link > div > img:first-child
-.roundbox-lt
-.roundbox-rt
-.roundbox-lb
-.roundbox-rb
+#userActivityGraph rect[data-items=""]
 .delete-resource-link
-a.contestParticipantCountLinkMargin > img
-.recent-actions > ul > li img[src*="hourglass"]
+.action-link img
+.non-decorated img[src$="tablesorter-bg.gif"]
 
 CSS
+#header > div:first-child > a::before {
+    content: "";
+    display: inline-block;
+    height: 65px;
+    width: 15px;
+    border-top-left-radius: 12px;
+    border-bottom-left-radius: 12px;
+    background-color: #fff;
```

Comment:
Are all of these needed?

Reply:
No, this is just a cosmetic fix. The thing is: I decided that by applying a negative to the logo, we change its colors, which changes the brand, which is not good... So, I just rounded the standard logo and added this `::before` element to make it symmetrical.

Default, rounded, rounded with extra `::before`:
![изображение](https://user-images.githubusercontent.com/38539949/145907341-e6c3c75e-74fd-43a8-a317-aafcbd08962a.png)
![изображение](https://user-images.githubusercontent.com/38539949/145907401-42872c1a-12c4-4ac7-b39e-81131c9b2f95.png)
![изображение](https://user-images.githubusercontent.com/38539949/145907425-892fefd8-0478-4fe8-b932-0c5948f73d2b.png)

Reply:
This does not brighten the page too much, since the logo is not large, and the rounded edges do not make it stand out from the design (all the edges on the site are rounded)

Reply:
> The thing is: I decided that by applying a negative to the logo, we change its colors, which changes the brand, which is not good...

It's common practice for dark reader to do that, so let's also do it here. This cosmetic change should be removed.

Reply:
Ok, this is really unnecessary, so I agree